nano.save

```diff
@@ -0,0 +1 @@
+
```

Comment:
some empty spaces at the modules completely counted from the quantum counting at the module deep background.
the dimmentional background of the space is the result of multiplication calculus which obtained the unity.